python/cuml/test/test_dbscan.py

```diff
@@ -73,6 +73,40 @@ def test_dbscan(datatype, use_handle, nrows, ncols,
         assert cu_labels.dtype == np.int64
 
 
+@pytest.mark.parametrize('max_mbytes_per_batch', [1e2, None])
```

Comment:
Does the quality test with `max_mbytes_per_batch=1e2` result in multiple batches during execution? Consider even lower memory value to test that.

Reply:
Yes, the quality test does, by a narrow margin (max potential size of the CSR matrix alone: `5000 * 5000 * 4 = 100M` + other arrays).
The unit test doesn't. Would you make the quality test a unit test? I don't want to lower the max_mbytes too much because then the stress test might take a while. Or I can use different max_mbytes for stress and unit/quality

Reply:
It would be nice to have a fast unit test which uses multiple batches. Using different max mbytes for unit / other would make sense.